### ISSUE TITLE
BUG: Cannot run pymc in windows 11 

### ISSUE THREAD

**Issue by JoshYang1977** [pymc-devs/pymc#6980]
### Describe the issue:

I was able to install pymc following [the instruction](https://github.com/pymc-devs/pymc/wiki/Installation-Guide-(Windows)). However, when I run the code, there are error messages that keep coming out.

### Reproduceable code example:

```python
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats
import pymc as pm
import pytensor.tensor as at


##### GENERATE DATA #####

d = stats.uniform.rvs(loc = 0, scale = 3, size = 100) 
true_theta = 0.336
true_epsilon = -0.01
rloss = np.exp(np.sqrt(d) * true_theta + true_epsilon)/(1 + np.exp(np.sqrt(d) * true_theta + true_epsilon))

##### SIMULATION #####
# MODEL BUILDING

with pm.Model() as model:
    
    theta = pm.Normal("theta", mu=0, sigma=1)
    epsilon = pm.Normal("epsilon", mu=0, sigma=1)
    phi = pm.Gamma("phi", alpha=0.01, beta=0.01)
    #mu = pm.math.invlogit(theta * np.sqrt(d) + epsilon)
    
    mu = pm.Deterministic("mu",pm.math.invlogit(theta * np.sqrt(d) + epsilon))
    a = pm.Deterministic("a", mu * phi)
    b = pm.Deterministic("b", (1 - mu)*phi)
    
with model:
    beta = pm.Beta("beta", alpha=a, beta=b, observed = rloss)

# MODEL RUN
with model:
    step = pm.Metropolis()
    trace = pm.sample(120000, step=step, chains=4)

theta_samples =  np.concatenate(trace.posterior.theta.data[:,1000:])
epsilon_samples =  np.concatenate(trace.posterior.epsilon.data[:,1000:])

##### PLOT RESULTS #####

ax = plt.subplot(211)


plt.hist(theta_samples, histtype='stepfilled', bins=25, alpha=0.7,
         label="posterior of $\\theta$", color="#A60628", density=True)
plt.vlines(true_theta, 0, 60, linestyle="--", color="black", label="True $\\theta$ (unknown)")
plt.legend(loc="upper right")

plt.title("Posterior distributions of $\\theta$, $\\epsilon$ unknowns")

ax = plt.subplot(212)

plt.hist(epsilon_samples, histtype='stepfilled', bins=25, alpha=0.7,
         label="posterior of $\\epsilon$", color="#348ABD", density=True)
plt.vlines(true_epsilon, 0, 60, linestyle="--", color="black", label="True $\\epsilon$ (unknown)")
plt.legend(loc="upper right")
```

### Error message:

```shell
UnicodeDecodeError: 'cp950' codec can't decode byte 0xe2 in position 2365: illegal multibyte sequence
```

### PyMC version information:

python 3.11.6
pytensor 2.17.3
pymc 5.9.1

**Comment by ricardoV94** [comment]
Those installation instructions are old. The new ones are here: https://www.pymc.io/projects/docs/en/stable/installation.html

**Comment by JoshYang1977** [comment]
@ricardoV94 thank you very much. I tried the newest[ installation](https://www.pymc.io/projects/docs/en/stable/installation.html) guide.

It gave me the same error message while eveything was installed.
I also tried to set PYTHONIOENCODING=utf-8 but did not fixt it.

**Comment by tvwenger** [comment]
What is the full traceback of the error @JoshYang1977? I'm not able to reproduce it.

**Comment by JoshYang1977** [comment]
@tvwenger Thank you very much.

I was able to fix this issue by doing these steps.

First I installed the latest pymc following the recommendation of @ricardoV94.
I ran the code and then the error code "UnicodeDecodeError: 'cp950'" came out.
Then I fixed the error by correcting `encoding="utf-8"` at related code and restarted the kernel.

Finally, I modified my code as follows because the code stopped 
![image](https://github.com/pymc-devs/pymc/assets/43284414/4cb57acd-f5da-4741-8ef0-ac3fe09b2102)

`trace = pm.sample(120000, step=step, cores=1, chains=1) `

I found it is an issue related to Windows system.

Thank you all for all wonderful answers so I can finally make my model to run.
